### DISCUSSION TITLE
How do I setup a keybind to open the NVIDIA Control Panel?

### DISCUSSION THREAD

**Discussion by Suvir0** [seerge/g-helper#1815] [Q&A] [answered]
Title, sorry if this is a stupid question, but I just couldn't figure it out. Thanks in advance!

**Comment by Kikimahe** [comment]
Hi,

I also tried to do a test, but I didn't succeed.

The Nvidia control panel (or other software) does not open with [the personalized shortcut.](https://github.com/seerge/g-helper/wiki/Power-user-settings#custom-hotkey-actions)

However the config.json file looks good:
` "fnf4": "custom",`
` "fnf4_custom": "C:\\Program Files\\WindowsApps\\NVIDIACorp.NVIDIAControlPanel_8.1.964.0_x64__56jybvy8sckqj\\nvcplui.exe"`


FN+F4 does not execute the shortcut.
[config.json](https://github.com/seerge/g-helper/files/13797549/config.json)
[log.txt](https://github.com/seerge/g-helper/files/13797550/log.txt)

**Reply by Kikimahe** [reply, marked as answer]
Got it ! :) 

This was not specified in the FAQ :) But in action, you must paste the program path into quotes **" "** !

![Capture d'écran 2023-12-30 165238](https://github.com/seerge/g-helper/assets/142881210/025a5cab-57fd-4f6e-ac5f-bc457ad9079f)



@CandiedArcher 
For the FN+C shortcut, in ACTION you must paste the Nvidia Control panel Path, with quotes : 
`"C:\Program Files\WindowsApps\NVIDIACorp.NVIDIAControlPanel_8.1.964.0_x64__56jybvy8sckqj\nvcplui.exe"`

(Your path may be different)

;-)

**Reply by Suvir0** [reply]
Thanks! That worked, I couldn't do it on the first try because I lacked the permissions. After changing the permissions I was able to use the hotkey flawlessly.

**Comment by seerge** [comment]
@CandiedArcher why do you want to open Nvidia control panel in a first place ? If to change Optimus state - there is a separate option somewhere in control panel settings, that will let you show a clickable optimus tray icon

**Reply by Suvir0** [reply]
Instead of swapping between ultimate and standard, I just wanted to keep it on optimized and swap between Optimus and dGPU quickly. So I wanted to make a hotkey (FN+C) to launch the NVIDIA Control Panel.

**Reply by seerge** [reply]
https://nvidia.custhelp.com/app/answers/detail/a_id/2542/~/how-do-enable-or-disable-the-nvidia-notification-tray-icon-on-the-windows
